src/event/messages/nez.ts

```diff
@@ -17,18 +19,33 @@ export const handleNezMessage = async (message: Message): Promise<void> => {
       .exec()
     const lastMessageTime: Date = userDoc?.lastMessageTime || new Date(0)
 
-    if (new Date().getTime() - lastMessageTime.getTime() < 1000 * 60) {
+    if (new Date().getTime() - lastMessageTime.getTime() < 60 * 1000) {
       console.log('actual time: ' + new Date())
       console.log('last message time: ' + lastMessageTime)
       console.warn('❌ ' + 'Two nose message in less than 60 sec')
       return
     }
 
-    await zenCountSchema.findOneAndUpdate(
-      { _id: message.author.id },
-      { $inc: { count: 1 }, lastMessageTime: new Date() },
-      { upsert: true, new: true },
-    )
+    const currentTime = new Date().getTime()
```

Comment:
Let's declare this const above and replace de date().getTime in the if statement above